apps/condo/domains/contact/schema/ExportContactsService.js

```diff
@@ -0,0 +1,63 @@
+const { EMPTY_DATA_EXPORT_ERROR } = require('@condo/domains/common/constants/errors')
+const { createExportFile } = require('@condo/domains/common/utils/createExportFile')
+const access = require('@condo/domains/contact/access/ExportContactsService')
+const { loadContactsForExcelExport } = require('@condo/domains/contact/utils/serverSchema')
+const { GQLCustomSchema } = require('@core/keystone/schema')
+const dayjs = require('dayjs')
+const timezone = require('dayjs/plugin/timezone')
+const utc = require('dayjs/plugin/utc')
+
+dayjs.extend(utc)
+dayjs.extend(timezone)
+
+const ExportContactsService = new GQLCustomSchema('ExportContactsService', {
+    types: [
+        {
+            access: true,
+            type: 'input ExportContactsToExcelInput { where: ContactWhereInput!, sortBy: [SortContactsBy!], timeZone: String! }',
```

Comment:
You do not need time zone here.

Reply:
I use here `ExportToExcelActionBar` with a hardcoded timezone. If this is a problem, it's possible to create the same sidebar without a timezone.
Should I create another one?

Reply:
At least you can make it optional - it's API and it requires smth that it will not use. 
The best solution will be to remove the timezone from all exports and to take the timezone from the organization together with translation language.

Reply:
@dkoviazin, fixed here e3d013d